inc/define.php

```diff
@@ -331,7 +331,7 @@
 
 
 $CFG_GLPI["socket_types"]                  = ['Computer','NetworkEquipment',
-    'Peripheral','Phone','Printer', 'PassiveDCEquipment'
+    'Peripheral','Phone','Printer', 'PassiveDCEquipment', 'Location'
```

Comment:
I am not confident with addition of `Location` here, in the middle of an asset types list. I am not sure that it will not induce unwanted side effects.